test/Feature/Atomics.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: %llvmas %s -o=%t.bc
```

Comment:
This tests seems to fail on older LLVM versions. I think the easiest course of action is to disable it there with
```
; REQUIRES: geq-llvm-3.7
```